### ISSUE TITLE
yaml load bug

### ISSUE THREAD

**Issue by nsheff** [pepkit/pepr#21]
the config printing of a nested list is nice, but it seems to choke on the microtest example:

the project config with multiple `pipeline_interfaces`, like this:
```
  # scripts (and accompanying pipeline config files) for submission.
  pipeline_interfaces:
    - ${CODE}/ATACseq/pipeline_interface.yaml
    - ${CODE}/open_pipelines/pipeline_interface.yaml
    - ${CODE}/dnameth_pipelines/pipeline_interface.yaml
    - ${CODE}/rnapipe/pipeline_interface.yaml
```

renders like this: 
```
> config(p)
PEP project object. Class: Config
 metadata:
    output_dir: /home/nsheff/microtest
    pipeline_interfaces: 
/home/nsheff/code/microtest/config/./././/home/nsheff/code///home/nsheff/code///home/nsheff/code///home/nsheff/code//ATACseq/open_pipelines/dnameth_pipelines/rnapipe/pipeline_interface.yaml/pipeline_interface.yaml/pipeline_interface.yaml/pipeline_interface.yaml
    sample_annotation: 
/home/nsheff/code/microtest/config/microtest_annotation.csv
```

but if I load it directly, it works:
```
> yaml::yaml.load_file(p@file)
$metadata
$metadata$output_dir
[1] "${HOME}/microtest"

$metadata$pipeline_interfaces
[1] "${CODE}/ATACseq/pipeline_interface.yaml"          
[2] "${CODE}/open_pipelines/pipeline_interface.yaml"   
[3] "${CODE}/dnameth_pipelines/pipeline_interface.yaml"
[4] "${CODE}/rnapipe/pipeline_interface.yaml"          
```

somehow the config is not getting loaded by `yaml::load_file()` correctly.

**Comment by nsheff** [comment]
I bet this has to do with `expandPath` changes. This used to work, I believe.

**Comment by nsheff** [comment]
@michalstolarczyk can you look at this, and add in a unit test to prevent future regressions?

**Comment by stolarczyk** [comment]
Can you check if the issue is fixed in your setting @nsheff ? aaa2a2e509290f8d184cc9bf36eca22527440ba9 resolved problems in my multiple-value test cases

**Comment by nsheff** [comment]
confirmed.
